lldb/source/Interpreter/CommandInterpreter.cpp

```diff
@@ -1887,7 +1887,8 @@ bool CommandInterpreter::HandleCommand(const char *command_line,
                                        CommandReturnObject &result,
                                        bool force_repeat_command) {
   std::string command_string(command_line);
-  std::string original_command_string(command_line);
+  std::string original_command_string(command_string);
+  std::string real_original_command_string(command_string);
```

Comment:
nit: does this makes the variable above not the real original command string :p ? may be we can find a better name or write a comment explaining the nuance between the 2

Reply:
This is just me being funny. I thought that if there is a std::string implementation that does CoW, then this would be more efficient, since the string data could be shared.